### PR TITLE
Fixes Hilbert Hotel and adds a beach condo

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

There was a plush that was mapped to a Skyrat folder that didnt exist anymore for the Apartment, given my internet is out i decided to just map in a beach condo as well.

image

image

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
Exit works
image

Yeah the rocks are a bit unsightly but uh - better than the void of space using the normal flags
image
image

  
</details>

## Changelog

:cl:
add: Hilbert Hotel - Adds a beach condo
fix: Apartment in Hilberts hotel regains its birb plush
/:cl:

### CODE REVIEW

File: modular_nova/modules/hotel_rooms/apartment.dmm [hunk 1]
```diff
@@ -350,7 +351,7 @@
 	},
 /area/misc/hilbertshotel)
 "Ib" = (
-/obj/item/toy/plush/skyrat/breakdancing_bird,
+/obj/effect/spawner/xmastree,
```
Comment (GoldenAlpharex) [inline modular_nova/modules/hotel_rooms/apartment.dmm:354]: I don't think this is a good idea.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just one quick change, rest looks good.

File: modular_nova/modules/hotel_rooms/apartment.dmm (hunk 1 above)
Comment (Zergspower) [reply modular_nova/modules/hotel_rooms/apartment.dmm:354]: The TG Apartment had a tree is why i added it, can remove it though if ya think

**Review by GoldenAlpharex — APPROVED** [review]
Should be good once CI passes!

**Pull request merged**